bindings_test.go

```diff
@@ -719,30 +719,75 @@ func testBindingArray(t *testing.T, bulk bool) {
 
 func TestBulkArrayBinding(t *testing.T) {
 	runDBTest(t, func(dbt *DBTest) {
-		dbt.mustExec(fmt.Sprintf("create or replace table %v (c1 integer, c2 string)", dbname))
+		dbt.mustExec(fmt.Sprintf("create or replace table %v (c1 integer, c2 string, c3 timestamp_ltz, c4 time)", dbname))
+		now := time.Now()
+		someTime := time.Date(1, time.January, 1, 12, 34, 56, 123456789, time.UTC)
 		numRows := 100000
 		intArr := make([]int, numRows)
 		strArr := make([]string, numRows)
+		ltzArr := make([]time.Time, numRows)
+		timeArr := make([]time.Time, numRows)
 		for i := 0; i < numRows; i++ {
 			intArr[i] = i
 			strArr[i] = "test" + strconv.Itoa(i)
+			ltzArr[i] = now
+			timeArr[i] = someTime
 		}
-		dbt.mustExec(fmt.Sprintf("insert into %v values (?, ?)", dbname), Array(&intArr), Array(&strArr))
+		dbt.mustExec(fmt.Sprintf("insert into %v values (?, ?, ?, ?)", dbname), Array(&intArr), Array(&strArr), Array(&ltzArr, TimestampLTZType), Array(&timeArr, TimeType))
 		rows := dbt.mustQuery("select * from " + dbname)
 		defer rows.Close()
 		cnt := 0
 		var i int
 		var s string
+		var ltz time.Time
+		var tt time.Time
 		for rows.Next() {
-			if err := rows.Scan(&i, &s); err != nil {
+			if err := rows.Scan(&i, &s, &ltz, &tt); err != nil {
 				t.Fatal(err)
 			}
-			if i != cnt {
-				t.Errorf("expected: %v, got: %v", cnt, i)
-			}
-			if exp := "test" + strconv.Itoa(cnt); s != exp {
-				t.Errorf("expected: %v, got: %v", exp, s)
+			assertEqualE(t, i, cnt)
+			assertEqualE(t, "test"+strconv.Itoa(cnt), s)
+			assertEqualE(t, ltz.UTC(), now.UTC())
+			assertEqualE(t, tt, someTime)
+			cnt++
+		}
+		if cnt != numRows {
+			t.Fatalf("expected %v rows, got %v", numRows, cnt)
+		}
+	})
+}
+
+func TestBulkArrayBindingTime(t *testing.T) {
+	runDBTest(t, func(dbt *DBTest) {
+		dbt.mustExec(fmt.Sprintf("create or replace table %v (s time(0), ms time(3), us time(6), ns time(9))", dbname))
+		someTimeWithSeconds := time.Date(1, time.January, 1, 1, 1, 1, 0, time.UTC)
+		someTimeWithMilliseconds := time.Date(1, time.January, 1, 2, 2, 2, 123000000, time.UTC)
+		someTimeWithMicroseconds := time.Date(1, time.January, 1, 3, 3, 3, 123456000, time.UTC)
```

Comment:
maybe it would be good to incorporate here consts like`time.Millisecond` etc?

Reply:
I don't like it, current form is more readable to me.